src/main/kotlin/net/lumalyte/lg/interaction/menus/guild/PeaceAgreementMenu.kt

```
@@ -371,12 +371,45 @@ class PeaceAgreementMenu(
+                player.sendMessage("§cYour guild is not currently at war with anyone.")
+                return
+            }
+            player.sendMessage("§6§l=== Active Wars ===")
+            activeWars.forEach { war ->
+                val enemyId = if (war.declaringGuildId == guild.id) war.defendingGuildId else war.declaringGuildId
+                val enemyGuild = guildService.getGuild(enemyId)
+                val enemyName = enemyGuild?.name ?: "Unknown"
+                player.sendMessage("§7- §c$enemyName")
+            }
+            player.sendMessage("§eUse §f/g peace propose <guild>§e to send a peace request.")
+        } catch (e: Exception) {
```

detekt (1.23.8)

[warning] The caught exception is too generic. Prefer catching specific exceptions to the case that is currently handled.

(detekt.exceptions.TooGenericExceptionCaught)